### ISSUE TITLE
tidy up nomenclature

### ISSUE THREAD

**Issue by ChristopherBarrington** [ChristopherBarrington/seuratvis#11]
percent_mitochondria : max_percent_mitochondria

**Comment by ChristopherBarrington** [comment]
```
reactiveValues(filtered_cell_set=NULL, n_cells=0, total_reads=0, median_reads_per_cell=0, median_genes_per_cell=0,
               min_genes_per_cell=NULL, max_genes_per_cell=NULL,
               min_expression_per_cell=NULL, max_expression_per_cell=NULL,
               max_percent_mitochondria=NULL) -> cell_filtering_data.reactions
```

could be 

```
reactiveValues(filtered_cell_set=NULL, n_cells=0, total_reads=0, median_reads_per_cell=0, median_genes_per_cell=0) -> cell_filtering_data.reactions,
reactiveValues(min_genes_per_cell=NULL, max_genes_per_cell=NULL,
               min_expression_per_cell=NULL, max_expression_per_cell=NULL,
               max_percent_mitochondria=NULL) -> cell_filtering_thresholds.reactions
```

**Comment by ChristopherBarrington** [comment]
can the server source scripts so the code is more isolated?

**Comment by ChristopherBarrington** [comment]
look into [modules](https://shiny.rstudio.com/articles/modules.html) and [splitting code](https://stackoverflow.com/questions/43002914/how-to-split-shiny-app-code-over-multiple-files-in-rstudio)

**Comment by ChristopherBarrington** [comment]
addressed by #24
